electroncash/commands.py

```diff
@@ -534,7 +537,12 @@ def _mktx(self, outputs, fee=None, change_addr=None, domain=None, nocheck=False,
             final_outputs.append((TYPE_ADDRESS, address, amount))
 
         coins = self.wallet.get_spendable_coins(domain, self.config)
-        tx = self.wallet.make_unsigned_transaction(coins, final_outputs, self.config, fee, change_addr)
+        if feerate is not None:
+            fee_per_kb = 1000 * PyDecimal(feerate)
+            fee_estimator = partial(SimpleConfig.estimate_fee_for_feerate, fee_per_kb)
```

Comment:
```suggestion
            fee_estimator = lambda size: SimpleConfig.estimate_fee_for_feerate(fee_per_kb, size)
```

Rationale: make the diff smaller :)